src/common/components/clusterConfiguration/SNOControlGroup.tsx

```diff
@@ -4,25 +4,29 @@ import { ClusterDetailsValues } from '../../../common/components/clusterWizard/t
 import { SingleNodeCheckbox } from '../ui';
 import { OpenshiftVersionOptionType } from '../../types';
 import SNODisclaimer from './SNODisclaimer';
-import { FeatureSupportLevelContext } from '../featureSupportLevels';
+import { useFeatureSupportLevel } from '../featureSupportLevels';
 
 type SNOControlGroupProps = {
   versions: OpenshiftVersionOptionType[];
   highAvailabilityMode: ClusterDetailsValues['highAvailabilityMode'];
+  isDisabled?: boolean;
```

Comment:
The idea was to use a global context to determine if a feature is diabled or not instead of props drilling. Is there no way to implement the same logic that determines the isDisabled property value in the FeatureSupportLevelContext provider?

Reply:
This time it's not about the feature but the flow. We have it enabled in the Create Cluster but not Edit flow. We can leverage FeatureSupportLevel for that but it somehow does not fit in my thinking.

Reply:
I know it's not ideal, it should be in a separate context, but for now it does eliminate the props drilling and is convenient to use

Reply:
We want to implement a separate context FeatureStateContext - does that fit more in your opinion? Question is if for now it can be in FeatureSupportLevel context

Reply:
I agree that disabling SNO due to editting the cluster is not really related to the SNO feature support so it should not be driven by the featuresupport context/hook.

Reply:
So the SNO can be disabled for 2 separate reasons (editting the cluster and feature not being supported in selected version)

Reply:
yes, that's why in OCM we return in getDisabledReason a different message for each case and it shows in a popover. Regarding the disabling in FeatureSupportLevel we want to refactor this